### ISSUE TITLE
Browser/Desktop: Accessibility - file input type lacks focus indication

### ISSUE THREAD

**Issue by patrickhlauke** [bitwarden/clients#3209]
### Steps To Reproduce

1. Navigate using the keyboard
2. `Tab`/`Shift+Tab` to set focus to a file input (e.g. in the "Create Send"/"Create New Send" form)

### Expected Result

The control needs a visible focus indication/outline.

### Actual Result

Outline is suppressed for this type of control.

### Screenshots or Videos

Animated gif for the popup/browser version, tabbing through the first few fields of the "Create New Send" form.
![bitwarden-focus-file-browser](https://user-images.githubusercontent.com/895831/182026113-7d4c4603-b363-4cef-b94c-35da7e727dcc.gif)

Animated gif for the desktop version, tabbing through the first few fields of the "Create Send" form.
![bitwarden-focus-file-desktop](https://user-images.githubusercontent.com/895831/182026116-5dabf240-400f-43e0-97d7-a8c771d950e7.gif)

### Operating System

Windows

### Operating System Version

10 Build 19044

### Web Browser

Chrome

### Build Version

2022.6.1 (browser) / 2022.6.2 (desktop)

**Comment by patrickhlauke** [comment]
Will submit a PR for this (currently battling with getting build environment working)
